src/hotspot/cpu/aarch64/assembler_aarch64.hpp

```diff
@@ -3663,6 +3663,18 @@ template<typename R, typename... Rx>
   INSN(sve_bic,  0b00, 0b0, 0b1);
 #undef INSN
 
+// SVE aliases
+#define INSN(ALIAS, REFERENT)                            \
+  void ALIAS(PRegister Pd, PRegister Pg, PRegister Pn) { \
+    REFERENT(Pd, Pg, Pn, Pg);                            \
+  }
+
+  INSN(sve_mov,  sve_and);  // Move predicates (zeroing); an alias of sve_and
+  INSN(sve_movs, sve_ands); // Move predicates (zeroing), setting the condition flags; an alias of sve_ands
+  INSN(sve_not,  sve_eor);  // Bitwise invert predicate; an alias of sve_eor
+  INSN(sve_nots, sve_eors); // Bitwise invert predicate, setting the condition flags; an alias of sve_eors
+#undef INSN
+
```

Comment:
These instructions are not used any more after the follow-up commit of using `EXT`. I suggest removing them.

Besides, could you also share the benchmark data after using `EXT`? I don't have >=256-bit SVE on hand and cannot test that. Thanks.